vm/isolated/isolated.go

```diff
@@ -304,6 +339,10 @@ func (inst *instance) Run(timeout time.Duration, stop <-chan bool, command strin
 		proxy := fmt.Sprintf("%v:127.0.0.1:%v", inst.forwardPort, inst.forwardPort)
 		args = append(args, "-R", proxy)
 	}
+	if inst.cfg.Pstore {
+		args = append(args, "-o", "ServerAliveInterval=6")
+		args = append(args, "-o", "ServerAliveCountMax=5")
```

Comment:
If we leave this, this need a comment as to why we do this only for pstore and why this is important.
But we need to be careful with the total timeout value. Need to figure out where it exactly it should fit within the hierarchy of all the timeouts we have, e.g.:
https://github.com/google/syzkaller/blob/master/vm/vm.go#L237-L247
https://github.com/google/syzkaller/blob/master/dashboard/config/upstream.sysctl#L4-L5
The current value of 30s is too low and won't allow most kernel debug facilities to kick in and provide an explanation.
But if ssh connection is broken, we should detect this as "no output" reliably and always without ssh keepalives, no? So why do we need this in the first place?

Reply:
When testing against the device without the timeouts, I noticed that Diagnose() was not getting called despite the device rebooting. Let me do a few more runs without the timeout and update this comment.

Reply:
Maybe it's not called in vm.MonitorExecution in some cases.
One case should be if it run for an hour and then rebooted without a crash for next run.